internal/rbd/migration.go

```diff
@@ -77,35 +77,43 @@ func parseMigrationVolID(vh string) (*migrationVolID, error) {
 // deleteMigratedVolume get rbd volume details from the migration volID
 // and delete the volume from the cluster, return err if there was an error on the process.
 func deleteMigratedVolume(ctx context.Context, parsedMigHandle *migrationVolID, cr *util.Credentials) error {
+	rv, err := genVolFromMigVolID(ctx, parsedMigHandle, cr)
+	if err != nil {
+		return err
+	}
+	err = deleteImage(ctx, rv, cr)
+	if err != nil {
+		log.ErrorLog(ctx, "failed to delete rbd image : %s, err: %v", rv.RbdImageName, err)
```

Comment:
log rv not just the image name

Reply:
@Madhu-1 you mean along with image name , log rv too?

Reply:
This change was not part of , actually I did that as an optimization. I can recheck again..

Reply:
Done.

Reply:
@humblec its not fixed yet.